### DISCUSSION TITLE
QField Windows GPS via COM-port

### DISCUSSION THREAD

**Discussion by diplopoda** [opengisch/QField#2297] [Q&A]
Hey, I'm testing QField on Windows 10 on my Surface Pro. Because there is no built-in GPS device I use btGPS to transmit GPS data from my phone (Pixel 6 Pro/Oneplus 7T Pro) to the Surface Pro. This works fine with several apps/software like QGIS etc.

It seems QField for Windows is not capable of getting the signal from the COM-port connected over bluetooth with the phone. Is there a workable solution to that or is it planned to make this possible? Thanks in advance. BR, Robert

**Comment by m-kuhn** [comment]
Hi @diplopoda

QField on Windows is currently lacking support for integrated as well as external GNSS. This is documented as issue https://github.com/opengisch/QField/issues/2038 . If you find a workaround or have additional information to add, please add it there.
Making this possible would be very welcome.
